Octokit.Tests.Integration/Clients/RepositoryActionsClientTests.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Octokit.Tests.Integration.Clients
+{
+    public class RepositoryActionsClientTests
+    {
```

Comment:
Related to previous point, it may be best to set up the integration tests don't make assumptions about the existing state. In this situation, it may be best to not test each API in isolation, but test the use of the API through several steps, e.g.:

 - create a secret, then retrieve it, then delete it
 - create a public key, then retrieve it, then delete it

Reply:
@shiftkey would it be worth testing the Get and GetAll repositories separately in this case since I wouldn't know if any values exist in those collections at all?